tests/testthat/test-plot.cluster_analysis.R

```diff
@@ -1,5 +1,6 @@
 test_that("`plot.see_cluster_analysis()` works", {
   library(parameters)
+  library(see)
```

Comment:
packages should be included with `if-require()` or `if-requireNamespace()` - why do we need _see_ here?